docs/advanced/platform-bom.md

```diff
@@ -17,8 +17,22 @@ In the case of the DGS Framework we have two different BOM definitions, the `gra
 and the `graphql-dgs-platform`. The latter only defines version alignment for the DGS modules while the first also
 defines versions for the dependencies of the DGS framework, such as Spring, Jackson, and Kotlin.
 
+## Using the Platform/BOM
 
-## Using the Platform/BOM?
+!!! warning
+    If you are using the [Spring Boot Dependency Management Plugin], please use the
```

Comment:
Maybe also add, that the graphql-java might be forced to whatever the plugin is using causing conflicts without this set up.